website/source/guides/operating-a-job/advanced-scheduling/preemption-service-batch.html.md

```diff
@@ -0,0 +1,447 @@
+---
+layout: "guides"
+page_title: "Preemption (Service and Batch Jobs)"
+sidebar_current: "guides-operating-a-job-preemption-service-batch"
+description: |-
+  The following guide walks the user through enabling and using preemption on
+  service and batch jobs in Nomad Enterprise (0.9.3 and above). 
+---
+
+# Preemption for Service and Batch Jobs
+
+~> **Enterprise Only!** This functionality only exists in Nomad Enterprise. This
+is not present in the open source version of Nomad.
+
+Starting with Nomad 0.9, every job in Nomad has a [priority][priority]
```

Comment:
We have always had priority associated with jobs, not new in 0.9. Replace with "Prior to Nomad 0.9, job priority in Nomad was used to process scheduling requests in priority order".